pkg/registry/garden/shoot/storage/tableconvertor.go

```diff
@@ -46,6 +46,7 @@ func newTableConvertor() rest.TableConvertor {
 			{Name: "Control", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["control"]},
 			{Name: "Nodes", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["nodes"]},
 			{Name: "System", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["system"]},
+			{Name: "ApiServer", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["apiserver"]},
```

Comment:
Rename to "APIServer".
Also, I would put it before "Control", so the order should be
APIServer, Control, Nodes, System (because that's the order how you write the conditions in the shoot status).